src/components/NavBar/__snapshots__/index.test.jsx.snap

```diff
@@ -60,13 +60,20 @@ exports[`navbar renders correctly 1`] = `
     >
       homePage.navbar.signIn
     </button>
-    <button
-      className="shadow-none mr-1 btn btn-outline-info"
-      disabled={false}
-      type="button"
+    <a
+      aria-current={null}
+      className="nav-link"
```

Comment:
is this a bootstrap class?

Reply:
I notice now that I was reviewing the index.test.jsx.snap. Don't take in consideration what I wrote